Error compiling fyne on Apple M1 arm64

Cannot compile fyne_demo on Apple M1 (if it matters).

I am following the procedure described at the [tour](https://developer.fyne.io/started/#run-demo), got an error message:

```
GO111MODULE=on go get fyne.io/fyne/cmd/fyne_demo
go: downloading github.com/go-gl/glfw/v3.3/glfw v0.0.0-20200625191551-73d3c3675aa3
# github.com/go-gl/gl/v3.1/gles2
go/pkg/mod/github.com/go-gl/gl@v0.0.0-20190320180904-bf2b1f2f34d7/v3.1/gles2/package.go:38:11: fatal error: 'KHR/khrplatform.h' file not found
 #include <KHR/khrplatform.h>
          ^~~~~~~~~~~~~~~~~~~
1 error generated.
```

### Device (please complete the following information):
 - OS: MacOS
 - Version: Big Sur 11.1
 - Go version: go version go1.16beta1 darwin/arm64
 - Fyne version: v1.4.3

--- a/internal/driver/glfw/glfw_core.go
+++ b/internal/driver/glfw/glfw_core.go
@@ -1,4 +1,4 @@
-// +build !gles,!arm,!arm64
+// +build !gles,!arm,!arm64 arm64,darwin
 
 package glfw
 

--- a/internal/driver/glfw/glfw_core.go
+++ b/internal/driver/glfw/glfw_core.go
@@ -1,4 +1,4 @@
-// +build !gles,!arm,!arm64 arm64,darwin
+// +build !gles,!arm,!arm64 darwin
 
 package glfw
 

--- a/internal/driver/glfw/glfw_es.go
+++ b/internal/driver/glfw/glfw_es.go
@@ -1,5 +1,5 @@
 // +build gles arm arm64
-// +build !arm64,!darwin
+// +build !arm64,darwin arm64,!darwin
 
 package glfw
 

--- a/internal/driver/glfw/glfw_es.go
+++ b/internal/driver/glfw/glfw_es.go
@@ -1,5 +1,5 @@
 // +build gles arm arm64
-// +build !arm64,darwin arm64,!darwin
+// +build !darwin
 
 package glfw
 

--- a/internal/driver/glfw/glfw_es.go
+++ b/internal/driver/glfw/glfw_es.go
@@ -1,4 +1,5 @@
 // +build gles arm arm64
+// +build !arm64,!darwin
 
 package glfw
 

--- a/internal/painter/gl/gl_core.go
+++ b/internal/painter/gl/gl_core.go
@@ -1,4 +1,4 @@
-// +build !gles,!arm,!arm64,!android,!ios,!mobile arm64,darwin
+// +build !gles,!arm,!arm64,!android,!ios,!mobile darwin
 
 package gl
 

--- a/internal/painter/gl/gl_core.go
+++ b/internal/painter/gl/gl_core.go
@@ -1,4 +1,4 @@
-// +build !gles,!arm,!android,!ios,!mobile amr64,darwin
+// +build !gles,!arm,!arm64,!android,!ios,!mobile arm64,darwin
 
 package gl
 

--- a/internal/painter/gl/gl_core.go
+++ b/internal/painter/gl/gl_core.go
@@ -1,4 +1,4 @@
-// +build !gles,!arm,!arm64,!android,!ios,!mobile
+// +build !gles,!arm,!android,!ios,!mobile amr64,darwin
 
 package gl
 

--- a/internal/painter/gl/gl_es.go
+++ b/internal/painter/gl/gl_es.go
@@ -1,6 +1,6 @@
 // +build gles arm arm64
 // +build !android,!ios,!mobile
-// +build !arm64,darwin arm64,!darwin
+// +build !darwin
 
 package gl
 

--- a/internal/painter/gl/gl_es.go
+++ b/internal/painter/gl/gl_es.go
@@ -1,4 +1,4 @@
-// +build gles arm arm64
+// +build gles arm arm64,!darwin
 // +build !android,!ios,!mobile
 
 package gl

--- a/internal/painter/gl/gl_es.go
+++ b/internal/painter/gl/gl_es.go
@@ -1,5 +1,6 @@
-// +build gles arm arm64,!darwin
+// +build gles arm arm64
 // +build !android,!ios,!mobile
+// +build !arm64,darwin arm64,!darwin
 
 package gl
 

